File: 240409/Next Level/next-level.py
person_id, level = input().split()

class game_info:
    def __init__(self, person_id = "codetree", level = 10):
        self.person_id = person_id
        self.level = level

    def print_info(self):
        print("user " + self.person_id + " lv", self.level)

games = game_info()
games.print_info()
games = game_info(person_id, level)
games.print_info()

File: 251231/Next Level/next-level.cpp
#include <iostream>
#include <string>

using namespace std;

class game_info 
{
    public:
        string person_id;
        string level;

        // 생성자 (기본값 포함)
        game_info(string pid = "codetree", string lvl = "10") {
            person_id = pid;
            level = lvl;
        }

        void print_info() {
            cout << "user " << person_id << " lv " << level << '\n';
        }
};

int main() 
{
    string person_id, level;
    cin >> person_id >> level;

    game_info games;
    games.print_info();

    games = game_info(person_id, level);
    games.print_info();

    return 0;
}
